### PR TITLE
Fix L3 RN50 tests accuracy

### PR DESCRIPTION
**Pull request by JanuszL**
- fix wrong path to TensorFlow data set
- make stop_at_epoch default to False in PyTorch plugin

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
Builds 615944

**Comment by JanuszL** [comment]
Rebuilds on 615959

**Comment by JanuszL** [comment]
Builds 616938

**Comment by JanuszL** [comment]
PyTorch works https://gitlab-master.nvidia.com/dl/dali/dali-gh-mirror/-/jobs/5367007
